apis/beacon/blobs/post_blobs.yaml

```diff
@@ -0,0 +1,54 @@
+post:
+  tags:
+    - Beacon
+    - ValidatorRequiredApi
+  summary: "Publish a signed blob."
+  operationId: "publishBlob"
+  description: |
+    Instructs the beacon node to broadcast a newly signed blob to the beacon network,
+    to be included in the beacon chain. A success response (20x) indicates that the blob
+    passed gossip validation and was successfully broadcast onto the network.
+    The beacon node is also expected to integrate the blob into state once it receives the 
```

Comment:
The description here needs to be changed. Blobs are not expected to be integrated into state.